### ISSUE TITLE
Names incorrect?

### ISSUE THREAD

**Issue by scheung38** [evheniy/react-app#1]
Part 1 of tutorial

mkdir app
cd app

In your tutorial, but where is this app directory and then things like webpack.config.js is in incorrect location? Hard to follow. Now I am not sure whether to follow or simply clone from your project to get it to work?

Why do we need two Dockerfiles? One at root and another one inside jenkins folder? Can we not use multiple FROM: statements in a single Dockerfile as per Docker 17.05?

**Comment by evheniy** [comment]
I got a lot of errors because of webpack new version.
I'll check it Today. Thank you!

**Comment by evheniy** [comment]
Hi.
You can just clone it, run npm install and run npm start.
If you need to run Jenkins CI with docker registry you can run container from jenkins dir. You can read more here: https://medium.com/@evheniybystrov/continuous-delivery-of-react-app-with-jenkins-and-docker-8a1ae1511b86
